floors/floors.go

```diff
@@ -0,0 +1,251 @@
+package floors
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"strings"
+
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/currency"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type Price struct {
+	FloorMin    float64
+	FloorMinCur string
+}
+
+const (
+	defaultDelimiter string = "|"
+	catchAll         string = "*"
+	skipRateMin      int    = 0
+	skipRateMax      int    = 100
+	modelWeightMax   int    = 100
+	modelWeightMin   int    = 1
+	enforceRateMin   int    = 0
+	enforceRateMax   int    = 100
+)
+
+// EnrichWithPriceFloors checks for floors enabled in account and request and selects floors data from dynamic fetched floors JSON if present
+// else selects floors JOSN from req.ext.prebid.floors and update request with selected floors details
+func EnrichWithPriceFloors(bidRequestWrapper *openrtb_ext.RequestWrapper, account config.Account, conversions currency.Conversions) []error {
+	err := []error{}
```

Comment:
Nitpick: This declaration seems unused. If you omit it, err will be declared on line 42.

Reply:
removed declaration